### PR TITLE
feat: pin std

### PR DESCRIPTION
**Pull request by lino-levan**
This PR is quite large. It:
- Bumps versions on everything and makes sure it's up to date
- Standardizes the language used in `@resource`
- Replaces all instances of `https://deno.land/std@version` with `$std/` in the examples
- Unfurls `$std/` at runtime to the `$std/`defined in the import map

### CODE REVIEW

File: data/benchmarking.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * @difficulty beginner
  * @tags cli
  * @run deno bench <url>
- * @resource {https://deno.land/manual@v1.29.1/tools/benchmarker} Deno: Benchmarker tool
+ * @resource {https://deno.land/manual@v1.30.0/tools/benchmarker} Manual: Benchmarker tool
```
Comment (kt3k) [inline data/benchmarking.ts:6]: How about removing this version number as well?

Comment (lino-levan) [reply data/benchmarking.ts:6]: We could do this, but the manual isn't stable yet. A few links actually had to be replaced with the closest thing I could find due to this. If that's okay with you, I'm game to remove the version number from the manual as well.

**Comment by kt3k** [comment]
The change itself looks good to me.

Very strangely this causes error in Deno Deploy for some reason. We are now investigating it (BTW I confirmed this works without issue locally with Deno cli)

**Comment by lino-levan** [comment]
@kt3k 
> Very strangely this causes error in Deno Deploy for some reason.

Any updates? Also, should we move ahead with removing the pin on the manual?

**Comment by lino-levan** [comment]
Fair enough, I was going to comment on this earlier but you didn't point it out. Import assertions are being re-spec-ed right? Looks like that fixed deploy compat?

**Comment by kt3k** [comment]
We haven't figured it out completely, but it looks like we can't import 'import map' json in Deno Deploy because of some bug (You can import non-'import map' json though)

(This is a minimial repro of the issue https://github.com/kt3k/deploy-import-assert-test

So we have to specify this mapping (`$std/ -> https://deno.land/std@0.175.0/`) directly in source code unfortunately..

**Review by kt3k — APPROVED** [review]
LGTM. Thanks for your contribution! @lino-levan

**Pull request merged**